app/tests/v2/test_base.py

```diff
@@ -0,0 +1,112 @@
+import os
+import unittest
+import json
+import pytest
+
+from app import create_app
+from app.database_config import test_init_db
+
+
+class BaseTestClass(unittest.TestCase):
+    """ This is the base class has test data """
+
+    def setUp(self):
+        """ Defines the test data """
+
+        self.app = create_app(config_name="testing")
+        self.client = self.app.test_client()
+        self.db = test_init_db()
+
+        self.data = {
+            "comment": "I am doing it",
+            "createdBy": 1,
+            "createdOn": "2018-12-12 15:45:07",
+            "images": "images",
+            "location": "naironi",
+            "status": "Draft",
+            "type": "Redflags",
+            "videos": "videos"
+        }
+
+        self.no_input = {
+        }
+
+        self.auth_signup = {
+            "firstname": "Anjichi",
+            "lastname": "Lupamo",
+            "othernames": "R",
+            "username": "Andela",
+            "email": "andela@andela.Kenya",
+            "phonenumber": "0724716026",
+            "password": "Eatlivecode"
+        }
+
+        self.no_comment = {
+            "createdOn": "2018-11-29 05:21:37",
+            "createdBy": "Norbert",
+            "location": "Mount Sinai",
+            "status": "There is a bush on fire",
+            "comment": "",
+            "id": 1
+        }
+
+        self.token_login = {
+            "username": "Andela",
+            "password": "Eatlivecode"
+        }
+
+        self.auth = {
```

Comment:
have your test data in a separate file

Reply:
I have separated the test data, by creating the test_base file. The only methods in the test_base file are for sign-up and login to generate token to authenticate the protected routes.